### PR TITLE
created RegisterAccountView

### PR DESCRIPTION
**Pull request by gina-wang-1021**
created the page where a user creates their account name, password and confirms password when they first uses the app.

### CODE REVIEW

File: lib/views/register_view.dart
Comment (dlion168) [inline lib/views/register_view.dart:1]: What is the difference between RegisterView and RegisterAccountView?

File: lib/views/register_account_view.dart [hunk 1]
```diff
@@ -0,0 +1,196 @@
...
+                  child: Padding(
+                    padding: EdgeInsets.fromLTRB(10.0, 0.0, 10.0, 6.0),
+                    child: Focus(
+                      onFocusChange: (hasFocus){
+                        setState(() {
+                          showPasswordHint = !showPasswordHint;
+                        });
+                      },
+                      child: TextField(
+                        //maxLength: 6,
```
Comment (dlion168) [inline lib/views/register_account_view.dart]: Delete the comments if they are unecessary.

File: lib/views/register_account_view.dart [hunk 2]
```diff
@@ -0,0 +1,196 @@
...
+                  width: 303.0,
+                  decoration: BoxDecoration(
+                      color: '7DDEF8'.toColor(),
+                      borderRadius: BorderRadius.circular(20),
+                      border: Border.all(width: 5.0, color: 'F5F5DC'.toColor())
+                  ),
+                  child: Padding(
+                    padding: EdgeInsets.fromLTRB(10.0, 0.0, 10.0, 6.0),
+                    child: TextField(
+                      //maxLength: 6,
```
Comment (dlion168) [inline lib/views/register_account_view.dart]: Dito. Delete unnecessary comment.

**Review by dlion168 — CHANGES_REQUESTED** [review]
Only some minor problems to fix & answer.
Looks good.

File: lib/views/register_view.dart
Comment (gina-wang-1021) [reply lib/views/register_view.dart:1]: RegisterView: where a user first chooses their grade and publisher
RegisterAccountView: where a user first sets their username, password and double checks password

I originally thought RegisterAccounView was RegisterView so edited the code in RevisterView, but then realized they were different views, so moved the code to another page. No major changes should be made in RegisterView.

File: lib/views/register_account_view.dart (hunk 1 above)
Comment (gina-wang-1021) [reply lib/views/register_account_view.dart]: deleted!

File: lib/views/register_account_view.dart (hunk 2 above)
Comment (gina-wang-1021) [reply lib/views/register_account_view.dart]: deleted comments in this file and the logInView file

File: lib/views/register_view.dart
Comment (dlion168) [reply lib/views/register_view.dart:1]: No problem, thank you for the explaination!

**Review by dlion168 — APPROVED** [review]
If possible, change LogInView to login_view in next commits.

**Pull request merged**